Scripts/routes.js

```diff
@@ -1,4 +1,3 @@
 page('/', homeController.index);
-page('/projects', projectController.index);
-page('/about', aboutController.index);
+page('/:section', sectionController.index);
```

Comment:
Hmmm, this does work, but it essentially captures everything after the `/`. You'll need to refactor this when you want to have more control over your routes at some point in the future.